CHANGELOG.md

```diff
@@ -11,6 +11,8 @@
 - Removed redundant `--seccomp-level` jailer parameter since it can be
   simply forwarded to the Firecracker executable using "end of command
   options" convention.
+- Renamed guest memory `region_end` to `last_addr`, changing the
+  behaviour for retreving the upper bound of a memory region.
```

Comment:
This changer shouldn't have user impac. I wouldn't mention it here.

Reply:
I thought that renaming public methods might bring some user impact. The rust-vmm/vm-memory crate should bump version because of the similar PR from there, which provides a breaking change. I can remove the specified changes from CHANGELOG, if it's not a similar case for Firecracker.

Reply:
I don't think we're in the same situation as `rust-vmm/vm-memory`. I would remove the changelog entry.

Reply:
`rust-vmm/vm-memory` repo exposes a crate and the code interface is also its public user-facing interface. `firecracker` is a binary with it's api as the user-facing interface.
I agree with @serban300 that this doesn't need to be in the changelog.

Reply:
Done, please take a look: https://github.com/firecracker-microvm/firecracker/commit/2cd48a4aa149da6c6c6605e9c8f3e341efb0fa59.